### DISCUSSION TITLE
generate graphql schema for postgres enum data types

### DISCUSSION THREAD

**Discussion by srikanthc-fl** [hasura/graphql-engine#9338] [Question]
We use postgres enums for predefined data values; but hasura does not generate the graphql schema for the enum types to refer. It will be helpful if hasura generates the schema for enum types as well. using the table as enum option has restriction on creating a foreign key relation and also does not support to have multi values to be set; or is there an option to set multi values using the table as enum option.

**Comment by adas98012** [comment]
@srikanthc-fl 

We have some support for enum values, here is the reference document:
https://hasura.io/docs/latest/schema/postgres/enums/#enums-in-the-hasura-graphql-engine
